### ISSUE TITLE
undesired prediction output

### ISSUE THREAD

**Issue by jaigsingla** [divamgupta/image-segmentation-keras#90]
I have chunk of 512x 512 images of urban area and ground truth of the same. ground truth is prepared in 0,1 and 2 labels. now using your training module , getting training accuracy of 96.43 using unet model but after executing predicts not getting desired output . Infact getting very odd output ..

**Comment by zdd0819** [comment]
> I have chunk of 512x 512 images of urban area and ground truth of the same. ground truth is prepared in 0,1 and 2 labels. now using your training module , getting training accuracy of 96.43 using unet model but after executing predicts not getting desired output . Infact getting very odd output .. further if i want to evaluate accuracy of predictions and ground truth then how to do that?

do u know how to evaluate it?
